### ISSUE TITLE
Output while using utils.plot_metrics

### ISSUE THREAD

**Issue by ghost** [catalyst-team/catalyst#517]
**Describe the bug**
I am using 

`
utils.plot_metrics(
    logdir=logdir, 
    # specify which metrics we want to plot
    metrics=["loss", "dice", 'lr', '_base/lr']
)
`
in google colab and the results are not plotted.

Can you suggest what can be the issue or any other way to get the same.

Thanks.

**Comment by ghost** [comment]
Screenshot is attached - 
![Screenshot from 2019-11-16 06-47-38](https://user-images.githubusercontent.com/46820227/68985543-43199a00-083d-11ea-956b-5f6198bf7f00.png)

**Comment by ghost** [comment]
As an alternative, I have now tried - 

`utils.plot_tensorboard_log (logdir = logdir,metrics = ['loss' , 'dice'] )
![by_batch](https://user-images.githubusercontent.com/46820227/68989492-c4892080-086d-11ea-865a-9637b322ff14.png)
`
It does plot the output but "by batch", when I use - 

`utils.plot_tensorboard_log (logdir = logdir,metrics = ['loss' , 'dice'] , step = 'num_epochs')
`
the output is null.

Can someone please suggest a way around to visualise the results?

Thanks.

**Comment by TezRomacH** [comment]
Try reloading both `configure_plotyly...` and `utils.plot_metrics`

**Comment by ghost** [comment]
Done it multiple times.
Can you tell a way to include epoch in the below - 

`utils.plot_tensorboard_log (logdir = logdir,metrics = ['loss' , 'dice'] , step = 'num_epochs')`

This can be the workaround.

**Comment by markson14** [comment]
> Done it multiple times.
> Can you tell a way to include epoch in the below -
> 
> `utils.plot_tensorboard_log (logdir = logdir,metrics = ['loss' , 'dice'] , step = 'num_epochs')`
> 
> This can be the workaround.

Have you fix this problem? I'm facing the same situation

**Comment by TezRomacH** [comment]
Okay, all right. Try to execute this snippet before importing catalyst/matplotlib/plotly.

**Comment by ghost** [comment]
@markson14  - did you tried the above as suggested ?

**Comment by markson14** [comment]
@PN12 I use tensorboard to open those logs instead of this func in the end.

**Comment by ghost** [comment]
@markson14  - were you able to visualise plots by epoch?

**Comment by markson14** [comment]
@PN12 No, by datetime or by step i think

**Comment by jchen42703** [comment]
You should be able to view your plots by epoch with tensorboard

**Comment by Scitator** [comment]
moved to contrib :)
